subxt/src/backend/mod.rs

```diff
@@ -325,9 +325,416 @@ pub enum TransactionStatus<Hash> {
+            fn request_raw<'a>(
+                &'a self,
+                method: &'a str,
+                params: Option<Box<serde_json::value::RawValue>>,
+            ) -> super::rpc::RawRpcFuture<'a, Box<serde_json::value::RawValue>> {
+                Box::pin(async move {
+                    match method {
+                        "state_getStorage" => {
+                            let mut data = self.data.lock().await;
+                            let params = params.map(|p| p.get().to_string());
+                            let rpc_params = jsonrpsee::types::Params::new(params.as_deref());
+                            let key: sp_core::Bytes = rpc_params.sequence().next().unwrap();
+                            let value = data.request.pop(key.0);
+                            value.map(|v| serde_json::value::RawValue::from_string(v).unwrap())
+                        }
+                        "chain_getBlockHash" => {
+                            let mut data = self.data.lock().await;
+                            let value = data.request.pop("chain_getBlockHash".into());
+                            value.map(|v| serde_json::value::RawValue::from_string(v).unwrap())
+                        }
+                        _ => todo!(),
+                    }
+                })
+            }
+
+            fn subscribe_raw<'a>(
+                &'a self,
+                _sub: &'a str,
+                _params: Option<Box<serde_json::value::RawValue>>,
+                _unsub: &'a str,
+            ) -> super::rpc::RawRpcFuture<'a, super::rpc::RawRpcSubscription> {
+                Box::pin(async {
+                    let mut data = self.data.lock().await;
+                    let values: RpcResult<Vec<RpcResult<Box<RawValue>>>> =
+                        data.subscription.read().await.map(|v| {
+                            v.into_iter()
+                                .map(|v| {
+                                    v.map(|v| serde_json::value::RawValue::from_string(v).unwrap())
+                                })
+                                .collect::<Vec<RpcResult<Box<RawValue>>>>()
+                        });
+                    values.map(|v| RawRpcSubscription {
+                        stream: futures::stream::iter(v).boxed(),
+                        id: Some("ID".to_string()),
+                    })
+                })
+            }
+        }
+        enum Conf {}
+        impl Config for Conf {
+            type Hash = crate::utils::H256;
+            type AccountId = crate::utils::AccountId32;
+            type Address = crate::utils::MultiAddress<Self::AccountId, ()>;
+            type Signature = crate::utils::MultiSignature;
+            type Hasher = crate::config::substrate::BlakeTwo256;
+            type Header = crate::config::substrate::SubstrateHeader<u32, Self::Hasher>;
+            type ExtrinsicParams = DefaultExtrinsicParams<Self>;
+
+            type AssetId = u32;
+        }
```

Comment:
Prob worth a comment for why this is necessary instead of just importing SubstrateConfig or whatever :)